LICENSE

```diff
@@ -1,202 +1,406 @@
+Except as otherwise noted, the content of this repository is licensed under the
+Creative Commons Attribution 4.0 License [1], and code and code samples are
+licensed under the Apache License, Version 2.0 [2].
+
+Copyright 2015 gRPC Authors and The Linux Foundation. All rights reserved.
```

Comment:
Note the starting copyright year of 2015, which is the year of grpc.io's [first blog entry](https://grpc.io/blog/principles/).